src/adaPets.js

```diff
@@ -9,29 +11,65 @@ const BASE_URL = 'http://localhost:3000/pets/';
 
 // Option functions.
 const listPets = () => {
-  // Fill out as part of Wave 1.
+   axios.get(BASE_URL)
+  .then((response) => {
+    setResult(response.data);
+  })
+  .catch((response) => {
+    setError(`List pets failed with a ${response.response} response!`);
+  })
+  .then(function () {
+    console.log("List pets request is complete!");
+  });
```

Comment:
Interesting that you aren't using a `finally` block here! Also, that you switched to the `function ()` syntax from arrow functions.

Nothing wrong, it's just visually a bit jarring. :)